README.md

```diff
@@ -2,41 +2,96 @@
 
 [![Build Status](https://travis-ci.org/Kinto/kinto-admin.svg)](https://travis-ci.org/Kinto/kinto-admin)
 
-A Web admin UI to manage Kinto collections.
+A Web admin UI to manage [Kinto](https://kinto.readthedocs.org/) collections.
+[Give it a go](http://kinto.github.io/kinto-admin/)!
 
 ## Installation
 
 NodeJS v4+ and npm 2.14+ should be instaled and available on your machine.
 
 ```bash
-$ npm install
+$ npm install kinto-admin --save
 ```
 
-## Development server
+Or if you want it to be globally available:
 
+```bash
+$ npm install -g kinto-admin
 ```
-$ npm start
+
+## Standalone local server
+
+The Web admin UI can be served locally using the `kinto-admin` executable,
+provided you pass it a valid JSON configuration file:
+
+```bash
+$ kinto-admin --config myconfig.json
+Using config at myconfig.json
+Listening at http://0.0.0.0:3000
+```
+
+You can specify the port to listen to using the `--port` or `-p` options:
+
+```bash
+$ kinto-admin --config myconfig.json -p 4000
+Using config at myconfig.json
+Listening at http://0.0.0.0:4000
+```
+
+## Configuration
+
+The required JSON configuration lists the collections to manage through the Web
+UI as well as their own individual configuration:
+
+```js
+// config.json
+{
+  // The name of a collection to manage
+  "tasks": {
+    // Human readable name
+    "name": "Todo tasks",
+    // Collection configuration object
+    "config": {
+      // The fields to render in list view
+      "displayFields": ["title", "done"],
+      // The JSON schema for this collection
+      "schema": {
+        "title": "Todo Tasks",
+        ...
+      }
+    }
+  },
+  // Another collection to manage
+  "articles": {
+    // ...
+  }
+}
 ```
```

Comment:
I found that writing json schema wasn't really simple to generate. But, the good news is that there is a generator for them, that you can find at http://jsonschema.net/#/. Maybe it's worth mentioning we can generate the schema there?